chpl/chpl-service/src/main/java/gov/healthit/chpl/app/statistics/SummaryStatistics.java

```diff
@@ -97,6 +91,9 @@ public static void main(final String[] args) throws Exception {
             while (endDate.compareTo(endDateCal.getTime()) >= 0) {
                 LOGGER.info("Getting csvRecord for start date " + startDateCal.getTime().toString() + " end date "
                         + endDateCal.getTime().toString());
+                System.out.println("Getting csvRecord for start date "
+                        + startDateCal.getTime().toString() + " end date "
```

Comment:
Why is this here? Why not use the LOGGER?

Reply:
File should never have been changed.  It has been reverted.